src/requests/addLinkToGroup.js

```diff
@@ -3,22 +3,16 @@
 import { makePostRequest } from '../request'
 import { AddLinkToGroupFailed } from '../errors'
 
-const PATH = '/v1/links/add'
+const PATH = '/v3/shorturls'
 
-const addLinkToGroup = async (url: string, TSID: string | number) => {
+const addLinkToGroup = async (url: string, groupId: string | number) => {
   const res = await makePostRequest(PATH, {
     url,
-    TSID,
+    groupId,
     domain: 'mish.gr',
     linkCreatorSetting: 'Simple'
   })
-  const [ linkResponse ] = res.body.LinkResponses
-
-  if (linkResponse.ErrorMessage && linkResponse.ErrorMessage !== '') {
```

Comment:
It used to be that if an error occurred the api would reply with a 200 status and an error message in the body. THey have fixed this is v3 so that the correct status is returned instead